apps/public-docsite-v9/.storybook/docs-root-v9.css

```diff
@@ -1,23 +1,18 @@
-/* remove the docs wrapper bg to let page bg show through */
-/* remove unnecessary padding now that theme switcher is not taking up space */
-#docs-root .sbdocs-wrapper {
-  background: transparent !important;
-  padding-top: 0;
-}
-
-/* sb-show-main is missing during page transitions causing a page shift */
-/* todo: cleanup once we no longer inherit docs-root */
-.sb-show-main.sb-main-fullscreen,
-.sb-main-fullscreen {
-  margin: 0;
-  padding: 0;
-  display: block;
-}
-
 /* remove loading overlay */
 .sb-preparing-story,
 .sb-preparing-docs,
 .sb-nopreview,
 .sb-errordisplay {
   display: none !important;
 }
+
+/* Hide props table controls, since they don't work in the React implementation */
```

Comment:
why is this needed, nothing like this exists in current master. lets focus only on web-component relate things and not touch react specific domain.

Reply:
@miroslavstastny added this, I’ll let him comment on the storybook changes

Reply:
This was added in #28424 by @levithomason. We have never got to a state to compare FUIR9 and FUIWC3 side by side. The two storybooks should be independent. Revert this change.